lib/pangea/utils/any_state_holder.dart

```diff
@@ -32,26 +32,32 @@ class PangeaAnyState {
     _layerLinkAndKeys.remove(transformTargetId);
   }
 
-  void openOverlay(OverlayEntry entry, BuildContext context) {
-    closeOverlay();
-    overlay = entry;
-    Overlay.of(context).insert(overlay!);
+  void openOverlay(
+    OverlayEntry entry,
+    BuildContext context, {
+    bool closePrevOverlay = true,
+  }) {
+    if (closePrevOverlay) {
+      closeOverlay();
+    }
+    entries.add(entry);
+    Overlay.of(context).insert(entry);
   }
 
   void closeOverlay() {
```

Comment:
Hello! I think it would be good to add a function / change this function that adds the ability to close a specific overlay. For instance, if a span card is open and an instruction card is open, with the instruction card on the top of the stack, and the user clicks the "ignore" button on the span card, it will close the instruction card instead, because it's at the top of the stack.